import.sql

```diff
@@ -130,14 +130,16 @@ CREATE TABLE IF NOT EXISTS `npwd_twitter_reports`
 CREATE TABLE IF NOT EXISTS `npwd_messages`
 (
     `id`              int(11)      NOT NULL AUTO_INCREMENT,
-    `message`         varchar(512) NOT NULL,
+    `message`         varchar(512) DEFAULT '',
```

Comment:
Is this change required? If so, we’ll need to alter existing tables within the server side of the phone on resource start.